apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -70,6 +71,7 @@ import { Notification } from './models/notification.model'
 import { SendNotificationResponse } from './models/sendNotification.response'
 import { notificationModuleConfig } from './notification.config'
 import { Defendant, DefendantService } from '../defendant'
+import { formatCourtIndictmentReadyForCourtEmailNotification } from '../../formatters/formatters'
```

Comment:
Consider updating the barrel file and import the formatter from there to be consistent with other formatter imports.